packages/SidePanel/src/components/Group/Group.styles.js

```diff
@@ -7,6 +7,9 @@ export const groupCSS = css`
   justify-content: flex-end;
   position: fixed;
   right: 0;
+  width: 100%;
+  z-index: ${props => props.zIndex};
```

Comment:
Need to set this to 100% or else, using % width on the side panels inside the group will not work correctly.

Z-index is added, was missing before.